js/driftME.js

```diff
@@ -148,14 +156,23 @@ function renderBox()
 // The predefined function to be called at the rate of 10 frames per second.
 function update()
 {
-	updateTimer();
```

Comment:
Why did you remove the updateTimer(); function call ? Please check and compare the changes before you generate any pull request.